### PR TITLE
Fix: icons viewbox

### PR DESCRIPTION
**Pull request by gabyzif**
## Summary

- This PR updates the viewbox of the icons to `0 0 20 20` 

image

Screenshot 2024-04-24 at 2 15 05 PM

Screenshot 2024-04-24 at 2 15 11 PM

- Closes [UNI-226](https://mparticle-eng.atlassian.net/browse/UNI-226)

[UNI-226]: https://mparticle-eng.atlassian.net/browse/UNI-226?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,3 +32,14 @@
 ## [1.9.6-dev.1](https://github.com/mParticle/aquarium/compare/v1.9.5...v1.9.6-dev.1) (2024-03-25)
 
 ## [1.8.3-dev.3](https://github.com/mParticle/aquarium/compare/v1.8.3-dev.2...v1.8.3-dev.3) (2024-03-20)
+
+### Realese a version
```
Comment (jared-dickman) [inline CHANGELOG.md:36]:
```suggestion
### Release a Production version
```

File: src/assets/svg/add.svg [hunk 2]
```diff
@@ -1,4 +1,3 @@
-<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">>
-    <path d="M12.5992 4.8002C12.5992 4.4702 12.3292 4.2002 11.9992 4.2002C11.6692 4.2002 11.3992 4.4702 11.3992 4.8002V11.4002H4.79922C4.46922 11.4002 4.19922 11.6702 4.19922 12.0002C4.19922 12.3302 4.46922 12.6002 4.79922 12.6002H11.3992V19.2002C11.3992 19.5302 11.6692 19.8002 11.9992 19.8002C12.3292 19.8002 12.5992 19.5302 12.5992 19.2002V12.6002H19.1992C19.5292 12.6002 19.7992 12.3302 19.7992 12.0002C19.7992 11.6702 19.5292 11.4002 19.1992 11.4002H12.5992V4.8002Z"
-          fill="currentColor"/>
+<svg width="20" height="20" viewBox="0 0 20 20"  xmlns="http://www.w3.org/2000/svg">
+<path d="M10.4744 4.30016C10.4744 4.03891 10.2606 3.82516 9.99935 3.82516C9.7381 3.82516 9.52436 4.03891 9.52436 4.30016V9.52517H4.29937C4.03812 9.52517 3.82437 9.73892 3.82437 10.0002C3.82437 10.2614 4.03812 10.4752 4.29937 10.4752H9.52436V15.7002C9.52436 15.9614 9.7381 16.1752 9.99935 16.1752C10.2606 16.1752 10.4744 15.9614 10.4744 15.7002V10.4752H15.6994C15.9606 10.4752 16.1744 10.2614 16.1744 10.0002C16.1744 9.73892 15.9606 9.52517 15.6994 9.52517H10.4744V4.30016Z" fill="currentColor"/>
```
Comment (jared-dickman) [inline src/assets/svg/add.svg:2]: the formatting was changed too and the indentation was lost? 
🤷 
we should go through your ide settings, there are also some sane defaults for Code Style in the indicative repo root `settings` directory

File: src/assets/svg/alicorn.svg [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
-          fill="currentColor"/>
-</svg>
\ No newline at end of file
+<svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" >
```
Comment (jared-dickman) [inline src/assets/svg/alicorn.svg:1]: oh, shoots. the `width` and `height` had been explicitly left out. it allows the svg to scale more naturally

File: src/components/general/Icon/Icon.stories.tsx [hunk 4]
```diff
@@ -14,7 +14,10 @@ export const  IconTable: React.FC<IIconProps> = ({ color = 'black', size = 'lg'
         justifyItems: 'center'
     }}>
         {(Object.keys(Icons) as Array<keyof typeof Icons>).map((iconName) => (
-            <Icon name={iconName} size={size} color={color} key={iconName} />
+            <div style={{display: "flex", flexDirection:"column", alignItems:"center", gap:"10px"}}>
```
Comment (jared-dickman) [inline src/components/general/Icon/Icon.stories.tsx:17]: **fwiw** theres a `<Flex/>` component from ant and a `<Center/>` component in the aquarium

unclear completely the pros and cons but they look cool and its slightly less code/syntax

File: src/components/general/Icon/Icon.stories.tsx [hunk 5]
```diff
@@ -14,7 +14,10 @@ export const  IconTable: React.FC<IIconProps> = ({ color = 'black', size = 'lg'
...
+                <Icon name={iconName} size={size} color={color} key={iconName} />
+                <p style={{fontFamily:"monospace"}}>{iconName}</p>
+            </div>
```
Comment (jared-dickman) [inline src/components/general/Icon/Icon.stories.tsx:20]: **praise** good idea adding the name under the icon in the [dictionary](https://www.chromatic.com/component?appId=65aec861fa4adbf0708aebd0&csfId=aquarium-general-icons&buildNumber=588&k=6628f8327f3dbb77dec7b2d3-1200px-interactive-true&h=5&b=-1)

File: src/components/general/Icon/Icon.stories.tsx (hunk 4 above)
Comment (jared-dickman) [reply src/components/general/Icon/Icon.stories.tsx:17]: **suggestion** use a style constant for the 10px, `SizeSm` or `PaddingSm`

File: CHANGELOG.md [hunk 6]
```diff
@@ -32,3 +32,14 @@
...
+
+1. Go to https://github.com/mParticle/aquarium/actions
+2. Click Release Aquarium
+3. Run workflow > main branch
```
Comment (jared-dickman) [inline CHANGELOG.md:40]: is this the right place for this? 
thought this file was autogenerated

Comment (tibuurcio) [reply CHANGELOG.md:40]: Yep, not the right place. That would be CONTRIBUTING.md

**Review by tibuurcio — APPROVED** [review]
Small docs change and we are good to go

**Review by tibuurcio — APPROVED** [review]

**Pull request merged**